fuse_examples/segmentation/unet.py

```diff
@@ -0,0 +1,113 @@
+
+""" Full assembly of the parts to form the complete network """
```

Comment:
Where did tou copy this code from?
What is the license? Should we add the link to the original code?